tests/pages/user-page.js

```diff
@@ -11,5 +12,7 @@ export default createPage({
     hover: triggerable('mouseenter'),
   },
 
+  courseProgressListItems: collection('[data-test-course-name]', CourseProgressListItem),
```

Comment:
In https://github.com/codecrafters-io/frontend/pull/1386#discussion_r1504967545,
did you mean this @rohitpaulk ?

Reply:
@ryan-gang yes, doesn't necessarily have to be a separate file, but that's the idea. Example: 

<img width="433" alt="Screenshot 2024-02-28 at 11 03 30" src="https://github.com/codecrafters-io/frontend/assets/3893573/12c0313e-6162-4bbd-8494-fa94bbc1d38c">

The usage you have here seems wrong btw, the collection selector should be something like `[data-course-progress-list-item]`